production-swarm/docker-stack.yml

```diff
@@ -3,20 +3,15 @@ services:
   jobservice:
     env_file:
       - ./rabbitmq.env
-      - ./postgres.env
     environment:
+      CAF_DATABASE_URL: "jdbc:postgresql://${JOBSERVICE_DB_HOST:-192.168.56.10}:${JOBSERVICE_DB_PORT:-5432}/jobservice"
```

Comment:
If we're going to have an `environment.sh` file then there's no point putting defaults in here as well - let's just update to `${JOBSERVICE_DB_HOST}` and leave out the default - same for other variables included in `environment.sh` file.